test_scripts/API/OpenMenuRPC/009_Open_Menu_RPC_without_HIM_response_GENERIC_ERROR.lua

```diff
@@ -0,0 +1,32 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0116-open-menu.md
+-- Description:
+-- In case:
+-- 1) Mobile application is set to appropriate HMI level and System Context (see table)

```

Comment:
@TMyza Since `table` is not described please provide the list of HMI levels and values for System Context
The same is applicable for `010`, `011`